.github/workflows/ui-accessibility.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: AWS UI and Accessibility Testing
+
+on: 
+  push:
+    branches: [ "main" ]
```

Comment:
I would only trigger it on workflow_dispatch, as the tests will always fail due to the length of time of the deployment

Reply:
Changed as requested.